repo/packages/C/confluent-kafka/10/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
sounds good. We'll keep it as is for this release. thanks for the quick responses.